decidim-initiatives/db/migrate/20181212155125_add_online_signature_enabled_to_initiative_type.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddOnlineSignatureEnabledToInitiativeType < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_initiatives_types, :online_signature_enabled, :boolean, null: false, default: true
```

Comment:
Isn't this going to make online signature enabled for all existing initiatives? Is that intended?

Reply:
This means that for all existing initiatives the select for the signature type will continue to show the three options (online, face-to-face, all). But the value stored in `Initiative.signature_type` is not modified.

I've realised about a missing edge case I'm not taking into account: if we update an existing `InitiativesType.online_signature_enabled` to `false`, we should update the `signature_type` of all the child initiatives still in the `created` state to `offline`, right?

Reply:
> This means that for all existing initiatives the select for the signature type will continue to show the three options (online, face-to-face, all). But the value stored in `Initiative.signature_type` is not modified.

I meant that shouldn't `online_signature_enabled` be false by default?

> I've realised about a missing edge case I'm not taking into account: if we update an existing `InitiativesType.online_signature_enabled` to `false`, we should update the `signature_type` of all the child initiatives still in the `created` state to `offline`, right?

Yes, it seems reasonable to do it!

Reply:
> I meant that shouldn't online_signature_enabled be false by default?

Sorry for asking again. Why should it be false by default?

I left it enabled because it preserves the current behavior, which is making all three options available. If we want to disable it for a existing `InitiativesType`, I think it's better to do it afterwards via updating the setting, otherwise  after running the migration is ran we may have initiatives with `online`/`any` set as `signature_type`, but belonging to a type with `online_signature_enabled` set to false.

Maybe I misunderstood something?

> Yes, it seems reasonable to do it!

Done in the last commit